stellar-captive-core/debian/stellar-captive-core.preinst

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+
+set -e
+
+if [ -f /lib/systemd/system/stellar-core.service ]; then
+  /bin/systemctl disable stellar-core
```

Comment:
Do we have guarantee that stellar-core is not running? Unless we do I think we should also stop it.

Reply:
There isn't a guarantee that stellar-core isn't running, but the stellar-core package doesn't start stellar-core.  That means that, if stellar-core is running the user did something to start it.